resources/views/requests/show.blade.php

```diff
@@ -82,16 +82,22 @@
                     {{ $torrentRequest->category->name }}
                 </span>
             </li>
-            <li class="request__type">
-                <span>
-                    {{ $torrentRequest->type->name }}
-                </span>
-            </li>
-            <li class="request__resolution">
-                <span>
-                    {{ $torrentRequest->resolution->name ?? 'No Res' }}
-                </span>
-            </li>
+            @if ($torrentRequest->type !== null)
+                <li class="request__type">
+                    <span>
+                        {{ $torrentRequest->type->name }}
```

Comment:
Instead of wrapping in condition why not use ?? 'Any' like already do in other places.